### PR TITLE
Build Raspberry Pi server page

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Build page at /raspberry-pi/server

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/raspberry-pi/server
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check against [copy doc ](https://docs.google.com/document/d/19PHu9Rh9JB60C_RGb63hXLPypO67Vr6T4bLDWNPjQYg/edit#)

## Issue / Card

Fixes [#8474](https://github.com/canonical-web-and-design/ubuntu.com/issues/8474)

### CODE REVIEW

**Comment by meralonso** [comment]
Hey @bethcollins92 it looks like the arm and docker logo are a bit stretched - can you put the right proportion, please? or I can re export assets for you.

File: templates/raspberry-pi/index.html [hunk 1]
```diff
@@ -0,0 +1,183 @@
+{% extends "raspberry-pi/_base_raspberry-pi.html" %}
+
+{% block title %}Ubuntu for Raspberry Pi{% endblock %}
+
+{% block meta_description %}The Raspberry Pi is an ARM computer for everybody. Perfect for teaching, coding, surfing the web or simply as a desktop for anyone, anywhere. The Raspberry Pi with Ubuntu is a gateway to the world of open source invention.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1XS8KyKQnQdgY9NJfxHMs_9ql4sPOTRj0yweunMPoOdM/edit#heading=h.alvyav8qzr5r{% endblock meta_copydoc %}
```
Comment (anthonydillon) [inline templates/raspberry-pi/index.html:7]: This is not the right doc

Comment (anthonydillon) [reply templates/raspberry-pi/index.html:7]: Should be: https://docs.google.com/document/d/1XS8KyKQnQdgY9NJfxHMs_9ql4sPOTRj0yweunMPoOdM/edit

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:

**Pull request merged**